src/btree/bt_debug.c

```diff
@@ -699,16 +701,38 @@ static int
 __debug_dsk_col_fix(WT_DBG *ds, const WT_PAGE_HEADER *dsk)
 {
     WT_BTREE *btree;
+    WT_CELL_UNPACK_KV unpack;
+    WT_COL_FIX_AUXILIARY_HEADER auxhdr;
     uint32_t i;
     uint8_t v;
 
     btree = S2BT(ds->session);
 
-    WT_FIX_FOREACH (btree, dsk, v, i) {
+    WT_RET(__wt_col_fix_read_auxheader(ds->session, dsk, &auxhdr));
+    switch (auxhdr.version) {
+    case WT_COL_FIX_VERSION_NIL:
+        WT_RET(ds->f(ds, "page version 0, no auxiliary data\n"));
+        break;
+    case WT_COL_FIX_VERSION_TS:
+        WT_RET(ds->f(ds, "page version 1, %" PRIu32 " time windows\n", auxhdr.entries));
+        break;
+    default:
+        WT_RET(ds->f(ds, "unknown page version %" PRIu32 "\n", auxhdr.version));
+        break;
+    }
+
+    WT_COL_FIX_FOREACH_BITS (btree, dsk, v, i) {
         WT_RET(ds->f(ds, "\t{"));
         WT_RET(__debug_hex_byte(ds, v));
         WT_RET(ds->f(ds, "}\n"));
     }
+
+    if (auxhdr.version == WT_COL_FIX_VERSION_TS) {
```

Comment:
It could be done, but it would be ugly. But more significantly, this is the debug_dsk version and it's probably better for it to print the disk contents as they appear (first the bitmap, then the time windows). The non-disk debug function does print them interleaved and that's the one that gets used most of the time.

Reply:
ok.